### ISSUE TITLE
Fcitx input method support

### ISSUE THREAD

**Issue by JunCEEE** [keshavbhatt/whatsie#17]
Thank you very much for developing this helpful software. It really saved me a lot of time. I just noticed that I could not switch to my fcitx input method in this whatsapp program. I'm wondering if it's possible to add the Fcitx input method support to this program? 

Thanks again for your efforts.

**Comment by keshavbhatt** [comment]
this has been discussed in https://github.com/keshavbhatt/whatsie/issues/15

**Comment by JunCEEE** [comment]
Thanks for referring to #15. I tried `snap refresh qt513 --edge`, but the problem stayed the same. Do you have any suggestions to check on my system?

Thanks in advance.

```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.1 LTS (fossa-squirtle X48)
Release:	20.04
Codename:	focal

fcitx version: 4.2.9.7
```
I have these lines in my `~/.profile`
```
export GTK_IM_MODULE=fcitx
export QT_IM_MODULE=fcitx
export XMODIFIERS=@im=fcitx
```

**Comment by keshavbhatt** [comment]
I have to give it a try on 20.04 when i will get my hand on system with the version of ubuntu installed. Thanks for reporting.

**Comment by JunCEEE** [comment]
It would be great. Thanks a lot.

**Comment by keshavbhatt** [comment]
The issue is fixed in 0483b030043f13fe048f575882ffb78d4177b479
You can refresh whatsie with `snap refresh whatsie --stable` to get the changes.
Feel free to reopen for further assistance.

**Comment by shakaran** [comment]
@JunCEEE, for Fcitx input, the maintained fork **Whatly** runs on current Qt6 WebEngine, where input-method handling (Fcitx included) is much improved over the old Qt5 build. https://github.com/shakaran/whatly
